build.yaml

```diff
@@ -495,7 +496,7 @@ common_cli:
     # we provide one by starting a webserver and pointing the tests
     # to look over there
     REPO_PATH=/results/omnibus/${TRIGGERED_BRANCH}/${DISTRIBUTION_NAME}
-    DOC_ROOT=/usr/share/nginx/html
+    DOC_ROOT=/var/www/html
```

Comment:
naive question as im less familiar with editing this file. Will this break any backward compatibility? or will all cases no use `/var/www/html`